daprdocs/content/en/reference/components-reference/supported-state-stores/setup-firestore.md

```diff
@@ -55,17 +57,22 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 
 | Field              | Required | Details | Example |
 |--------------------|:--------:|---------|---------|
-| type               | Y        | The credentials type | `"serviceaccount"`
 | project_id         | Y        | The ID of the GCP project to use | `"project-id"`
-| private_key_id     | Y        | The ID of the prvate key to use  | `"private-key-id"`
-| client_email       | Y        | The email address for the client | `"eample@example.com"`
-| client_id          | Y        | The client id value to use for authentication | `"client-id"`
-| auth_uri           | Y        | The authentication URI to use | `"https://accounts.google.com/o/oauth2/auth"`
-| token_uri          | Y        | The token URI to query for Auth token | `"https://oauth2.googleapis.com/token"`
-| auth_provider_x509_cert_url | Y | The auth provider certificate URL | `"https://www.googleapis.com/oauth2/v1/certs"`
-| client_x509_cert_url | Y      | The client certificate URL | `"https://www.googleapis.com/robot/v1/metadata/x509/x"`
+| endpoint       | N  | GCP endpoint for the component to use. Only used for local development with, for example, [GCP Datastore Emulator](https://cloud.google.com/datastore/docs/tools/datastore-emulator). The `endpoint` is unncessary when running against the GCP production API. | `"localhost:8432"`
+| private_key_id     | N        | The ID of the prvate key to use  | `"private-key-id"`
+| client_email       | N        | The email address for the client | `"eample@example.com"`
+| client_id          | N        | The client id value to use for authentication | `"client-id"`
+| auth_uri           | N        | The authentication URI to use | `"https://accounts.google.com/o/oauth2/auth"`
+| token_uri          | N        | The token URI to query for Auth token | `"https://oauth2.googleapis.com/token"`
+| auth_provider_x509_cert_url | N | The auth provider certificate URL | `"https://www.googleapis.com/oauth2/v1/certs"`
+| client_x509_cert_url | N      | The client certificate URL | `"https://www.googleapis.com/robot/v1/metadata/x509/x"`
 | entity_kind          | N      | The entity name in Filestore. Defaults to `"DaprState"` | `"DaprState"`
 | noindex              | N      | Whether to disable indexing of state entities. Use this setting if you encounter Firestore index size limitations. Defaults to `"false"` | `"true"`
+| type                 | N       | **DEPRECATED** The credentials type | `"serviceaccount"`
+
+
+## GCP Credentials
+The GCP Firestore component uses the GCP Go Client Libraries and by default it authenticates using **Application Default Credentials** as explained in the [Authenticate to GCP Cloud services using client libraries](https://cloud.google.com/docs/authentication/client-libraries) 
```

Comment:
```suggestion
Since the GCP Firestore component uses the GCP Go Client Libraries, by default it authenticates using **Application Default Credentials**. This is explained in the [Authenticate to GCP Cloud services using client libraries](https://cloud.google.com/docs/authentication/client-libraries) guide.
```